creator-cards/jean-emmanuel-baillat.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Jean-Emmanuel Baillat",
+  "type": "CREATOR",
+  "tags": "C#, .NET MAUI, Xamarin.Forms, Beginner, French",
+  "countryCode": "fr",
+  "github": "https://www.github.com/Kapusch",
+  "channels": [
+    {
+      "name": "Blog",
+      "url": "https://www.mauicestclair.fr/"
```

Comment:
Please also link the English blog here, then we are good!

Reply:
Sure, done!